src/components/Footer.module.css

```diff
@@ -2,8 +2,7 @@
   background-color: lightgray;
   height: 250px;
   display: flex;
-  flex-direction: row;
-  padding-top: 3%;
+  padding: 3% 11% 0 11%;
```

Comment:
again, padding being measured in percentage seems odd.  

In other details you don't need the second 11% (padding: 3% 11% 0; means the same thing)